modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItTemporalIndexTest.java

```diff
@@ -68,12 +68,12 @@ static void initTestData() {
                 .app("CREATE TABLE DATE1 (pk DATE, val DATE, PRIMARY KEY USING SORTED (pk));").nl()
                 .app("CREATE TABLE DATE2 (pk DATE, val DATE, PRIMARY KEY USING HASH (pk));").nl()
                 .app("CREATE TABLE TIME1 (pk TIME, val TIME, PRIMARY KEY USING SORTED (pk));").nl()
-                .app("CREATE TABLE TIME2 (pk TIME, val TIME, PRIMARY KEY USING HASH (pk));").nl()
+                .app("CREATE TABLE TIME2 (pk TIME(6), val TIME, PRIMARY KEY USING HASH (pk));").nl()
```

Comment:
why do we need such a change ?

Reply:
I added this for consistency, for example if a dynamic parameter had a default precision of 0.
But you're right, we don't need it.
Reverted this change, thanks.